benchmarks/src/main.rs

```diff
@@ -61,8 +49,8 @@ fn get_nth_percentile(durations: &[Duration], n: f64) -> Duration {
     durations[std::cmp::max(index, 1) - 1]
 }
 
-fn score_test(durations: Vec<Duration>, name: &str, task_size: f64, more_info: bool) -> f64 {
-    let median = get_nth_percentile(&durations, (*MAX_ITERATIONS / 2) as f64);
+fn score_test(mut durations: Vec<Duration>, name: &str, task_size: f64, more_info: bool) -> f64 {
```

Comment:
Is this `mut` necessary? It seemed to pass the linter but I don't see anywhere durations is mutated.

Reply:
Oops, not, it's not. I added it when sorting to the vector inside the method, but then later realized it was sorted elsewhere, but I forgot to remove the `mut` here. I'll do that now.